src/containers/coursePageDetails/index.jsx

```diff
@@ -1,27 +1,55 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
 
 import StarRatings from 'react-star-ratings';
 import { withTranslation } from 'react-i18next';
 
-import { getCourseDetails } from '../../services/course';
+import { getCourseDetails, subscribeCourse, subscribeCourseCheck } from '../../services/course';
 import { Spinner } from '../../components/spinner';
 
 import styles from './styles.module.scss';
 
+
 class coursePageDetails extends Component {
   state = {
     course: {},
```

Comment:
Why are you using camelCase for classes?

Reply:
As I remember, we decided to use camelCase in styles

Reply:
This is the convention that we capitalize classes.

Reply:
`CoursePageDetails`